index.html

```diff
@@ -110,9 +112,7 @@ <h4>CG Ignite </h4>
               </p>
             </div>
           </div>
-          <u>
-            <h5>SEE THE WHOLE PROGRAM </h5>
-          </u>
+            <h5><u>SEE THE WHOLE PROGRAM</u></h5>
```

Comment:
- [ ] Please make sure to center cards elements in the `main program` section as provided by the original design 

Current design            |  original design
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/163588081-c00c829f-9153-45a6-b812-bab2f80640cf.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/163588070-6daf9d20-c15c-4cf5-ba7b-d72f0ed2bf07.PNG)